openIMIS/openIMIS/settings.py

```diff
@@ -136,7 +136,14 @@ def SITE_URL():
     ALLOWED_HOSTS = ["*"]
 
 # TEST_WITHOUT_MIGRATIONS_COMMAND = 'django_nose.management.commands.test.Command'
-# TEST_RUNNER = 'core.test_utils.UnManagedModelTestRunner'
+TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
+
+# Tell nose to measure coverage on the 'foo' and 'bar' apps
+NOSE_ARGS = [
+    '--with-coverage',
+    '--cover-package=foo,bar',
```

Comment:
Is `foo,bar`a correct input for this arg? If it should represent installed openIMIS modules then we can set up this value programmatically basing on the content of openimis.json